<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>185</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>383</width>
    <height>185</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>383</width>
    <height>185</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>163lrc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="idEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>113</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>你可以从url中获得</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>11</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>下载</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="musicButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>单曲</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="playlistButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>歌单</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="albumButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>专辑</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>这是:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>11</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>40</y>
      <width>151</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>状态</string>
    </property>
    <widget class="QLabel" name="statusinfolabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>状态
信息</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>115</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>总:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>已:</string>
    </property>
   </widget>
   <widget class="QLabel" name="alllabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>45</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="finishedlabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>45</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>206</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>30</y>
       <width>45</width>
       <height>10</height>
      </rect>
     </property>
     <property name="text">
      <string>歌词样式:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>60</y>
       <width>45</width>
       <height>10</height>
      </rect>
     </property>
     <property name="text">
      <string>延时MSec:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>80</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>文件名格式:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="styleBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>26</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>[0]将外文歌词和翻译同行(*默认)
[1]将外文歌词和翻译分行，按延时Msec参数延时</string>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="delayBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>56</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>分行处理时两行的时间差
默认为100，即1秒</string>
     </property>
     <property name="maximum">
      <number>3000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QComboBox" name="filenameBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>82</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="toolTip">
      <string>歌词文件的文件名格式，支持自定义</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="piclabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>7</y>
      <width>84</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/new/png/Built_with_Qt_RGB_logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>https://github.com/ludoux/163lrc</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
